src/sage/algebras/lie_algebras/abelian.py

```diff
@@ -159,7 +159,7 @@ def dimension(self):
         """
         return infinity
 
-    def is_abelian(self):
+    def is_abelian(self) -> bool:
```

Comment:
```suggestion
    def is_abelian(self) -> Literal[True]:
```

And add `from typing import Literal` at the top.

Reply:
I would keep that for later. We are very far from such a level of detail, and in fact from doing anything useful with annotations.